awx/main/models/credential/injectors.py

```diff
@@ -31,7 +31,7 @@ def gce(cred, env, private_data_dir):
     f = os.fdopen(handle, 'w')
     json.dump(json_cred, f, indent=2)
     f.close()
-    os.chmod(path, stat.S_IRUSR | stat.S_IWUSR)
+    os.chmod(path, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IWGRP)
```

Comment:
Seeing this repeated so many times, it makes it seem obvious that it should become a shared constant, and an easy location for this is `awx.main.constants`. We might have multiple variety of file permissions for executable files, and some others.

Reply:
+1